android/src/main/kotlin/com/github/picture2pc/android/ui/main/MainScreen/elements/Clients.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.picture2pc.android.ui.main.MainScreen.elements
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.github.picture2pc.android.viewmodel.MainScreenViewModels.ClientsViewModel
+import org.koin.compose.rememberKoinInject
+
+@Composable
+fun Clients(modifier: Modifier){
+    val viewModel: ClientsViewModel = rememberKoinInject()
```

Comment:
Should be passed as parameter with injection being the default value to enable calling this composable without koin